src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowSkeleton.java

```diff
@@ -35,7 +35,7 @@ public AbstractWindowSkeleton(final String resource)
      * @param id     Button ID.
      * @param action Consumer with the action to be performed.
      */
-    public final void registerButton(String id, Runnable action)
+    public final void registerButton(final String id, final Runnable action)
     {
         registerButton(id, (button) -> action.run());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "button" parameter (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1611)